horace_core/sqw/@sqw/private/do_sqw_eval_average_filebacked_.m

```diff
@@ -40,17 +41,14 @@
     % Assign each pixel's signal to bin average, variance set to zero
     pix_chunk.signal = repelem(ave_signal, npix_chunk);
     pix_chunk.variance = 0;
+    data = pix_chunk.data;
+    pix.data_range = pix_chunk.pix_minmax_ranges(data, ...
+        pix.data_range);
 
-    pix_chunk = pix_chunk.reset_changed_coord_range({'signal', 'variance'});
-
-    wout.pix.data_range = minmax_ranges(pix_chunk.data_range, ...
-                                        wout.pix.data_range);
-
-    wout.pix = wout.pix.format_dump_data(pix_chunk.data);
-
+    pix = pix.format_dump_data(data);
 end
-
-wout.pix = wout.pix.finish_dump();
+wout.pix = pix.finish_dump();
+%TODO: IT MUST go with apply
```

Comment:
can you clarify what this means please

Reply:
I am modifyng code appropriately in the following ticket I am doing now -- this piece of code should be modified using apply